### ISSUE TITLE
Algorithms are not exported

### ISSUE THREAD

**Issue by nantonel** [kul-forbes/ProximalAlgorithms.jl#23]
Writing `ProximalAlgorithms.ForwardBackward()` is quite long.

Is there a particular why the constructors are not exported?

**Comment by lostella** [comment]
I'm not sure, I've been leaning towards being more parsimonious with `export`s lately.

Note that one can always explicitly bring module components into the current scope:

```
using ProximalAlgorithms: ForwardBackward, DouglasRachford
```

**Comment by nantonel** [comment]
Well I think this is too long:
```julia
@minimize ls(A*x-b) st norm(x) == 2.0 with ProximalAlgorithms.ForwardBackward(fast=true)
```
Are you suggesting one has to do:
```julia
using StructuredOptimization
using ProximalAlgorithms: ForwardBackward
```
every time he wants to use the package and play with the solver parameters?

Could we at least export these in StructuredOptimization?
```
import ProximalAlgorithms:ForwardBackward
export ForwardBackward
```
This would mean we can add these in the documentation of StructOpt, making it more complete.

**Comment by lostella** [comment]
> Could we at least export these in StructuredOptimization?

Yeah that we could do

> This would mean we can add these in the documentation of StructOpt

Would it be possible to do that by relying on the docstrings from ProximalAlgorithms? If yes, then why not

**Comment by nantonel** [comment]
> Would it be possible to do that by relying on the docstrings from ProximalAlgorithms? If yes, then why not

Apparently it's possible, just tried it out. I can do that if you want. Can you grant me access to your fork of StructOpt?

**Comment by nantonel** [comment]
Solved in [StructuredOptimization#21](https://github.com/kul-forbes/StructuredOptimization.jl/pull/21)
